.gitignore

```diff
@@ -13,6 +13,5 @@
 *.log
 *.patch
 /.vscode
-/package-lock.json
```

Comment:
Is this file NOT supposed to be excluded in your commits? This file was ignored earlier, what is the value of including it for now?

Reply:
The file is intended to be committed into version control. https://docs.npmjs.com/files/package-lock.json

Reply:
Thanks, this is helpful